fob/roadtrip/side/side2.sqf

```diff
@@ -0,0 +1,41 @@
+﻿// ------ Side satellite -----
+// Author : [LM]Cheitan
+// Team   : La Muerta
+
+_objets = ["Land_WaterBottle_01_cap_F","Land_TorqueWrench_01_F","Land_BoreSighter_01_F","Land_TinContainer_F","Land_Sleeping_bag_folded_F","Land_FoodContainer_01_F"];
+
+_marker_roadtrip_side2 = createMarker ["marker_roadtrip_side2", [26293.1,25941.1]];
+"marker_roadtrip_side2" setMarkerShape "ellipse";
+"marker_roadtrip_side2" setMarkerSize [2500,700];
+"marker_roadtrip_side2" setMarkerDir 200;
+"marker_roadtrip_side2" setMarkerBrush "Horizontal";
+"marker_roadtrip_side2" setMarkerAlpha 0.5;
+"marker_roadtrip_side2" setMarkerColor "ColorOrange";
+
+_centre = [27940.6,25293.7,0];
+
+{
+	//Créer l'objet
+	_v = createVehicle [_x, [0,0,0], [], 0, "NONE"];
```

Comment:
_vehicule = createVehicle [_x, [0,0,0], [], 0, "NONE"];

Reply:
De même, à quoi correspond _x ?
**Réponse :** c'est l'itérateur du langage.
Note : Préparer des vomibag pour les prochains code review ...